### PR TITLE
Connecting database to react

### PR DESCRIPTION
**Pull request by rslaughter12**
In this commit I have created separate files for the two queries that are happening so far, deleted the original queries.js file. 
Then changed around to have the category tabs to be populated via data base. Prettier was run in all files that were edited.

### CODE REVIEW

File: client/src/components/Navbar.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import {
 } from '@material-ui/core';
 import SearchIcon from '@material-ui/icons/Search';
 import { NavLink } from 'react-router-dom';
+import LearnifyLogo from '../images/learnify.png';
```
Comment (sarah-jensen) [inline client/src/components/Navbar.js:14]: I don't see the images folder?

File: client/src/components/Carousel.js [hunk 2]
```diff
@@ -49,24 +49,29 @@ function Carousel() {
...
+      image:
+        'https://c0.wallpaperflare.com/preview/272/750/376/business-laptop-office-computer.jpg',
     },
     {
-      title: 'Item 2',
-      description: 'Description for Item 2',
-      image: 'item2.jpg',
+      title: 'Tutorials',
+      description:
+        'Lernify contains a wide range of tutorials for you to learn from!',
```
Comment (sarah-jensen) [inline client/src/components/Carousel.js]: typo on "Lernify" --> "Learnify"

**Review by sarah-jensen — COMMENTED** [review]
I'm not sure where the code is for the timing of the carousel switching (whether it's in our repo or coming from MUI), but if you click ahead to the next image toward the end of the timeout period it switches almost right away to the next one (like it progresses every 5 seconds, but the timer does not reset if manually progressed to the next card).

Carousel.js: line 60 typo (Lernify-->Learnify)

**Review by sarah-jensen — COMMENTED** [review]
maybe consider the media card components from MUI for the cards with images? they're helpful for having consistent style and sizing for images/links/text across many cards.

https://v4.mui.com/components/cards/#media

**Comment by UserOlena** [comment]
There are a lot of errors in the browser's console on the Home page loading

**Review by sendusyourbones — APPROVED** [review]
Nice work! Love the carousel!

**Pull request merged**